### PR TITLE
Ledger Live integration

### PR DESCRIPTION
**Pull request by ckoopmann**
## **Summary of Changes**

&nbsp;

## **Test Data or Screenshots**

&nbsp;

###### _By submitting this pull request, you are confirming the following to be true:_

- I have reviewed the [Contribution Guidelines](https://github.com/IndexCoop/index-app/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `IndexCoop/index-app`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "en": "DeFi doesn’t have to be complicated. Unlock powerful sector, leverage and yield-earning strategies with our simple index tokens."
+    }
+  },
+  "permissions": [
+    {
+      "method": "*"
+    }
+  ],
+  "domains": ["https://*"]
+}
```
Comment (janndriessen) [inline manifest.json:43]: I guess this manifest needs to be different from the regular one? Otherwise, it would have to be somewhat "merged" with the one under `/public`

File: package.json [hunk 2]
```diff
@@ -24,18 +25,19 @@
...
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-app-rewired start",
     "hardhat": "npx hardhat node",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test  --transformIgnorePatterns 'node_modules/(?!@wagmi)/'",
+    "eject": "react-app-rewired eject",
```
Comment (janndriessen) [inline package.json:40]: If this is changed for the whole project, we should watch for side effects on regular vercel hosting. On the other hand we hopefully move away from create-react-app soon anyway.

File: src/utils/wagmi.ts [hunk 3]
```diff
@@ -48,7 +51,9 @@ const connectors = connectorsForWallets([
       trustWallet({ chains }),
     ],
   },
-])
+])().concat([ledgerConnector])
+
+console.log("connectors:", connectors())
```
Comment (janndriessen) [inline src/utils/wagmi.ts:56]: Can be removed, unless it makes sense to keep it for now for testing.

File: vercel.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
       "headers": [
         {
           "key": "Content-Security-Policy",
-          "value": "frame-ancestors https://app.safe.global"
+          "value": "frame-ancestors https://app.safe.global https://eth-dapp-browser-ne49wqncp-ledgerhq.vercel.app"
```
Comment (janndriessen) [inline vercel.json:8]: I guess this should be the production value? Or at least at the production url additionally?

**Review by janndriessen — APPROVED** [review]
LGTM! Some minor comments to consider.

Also, please run `yarn lint:fix` before merging. ;)

File: manifest.json (hunk 1 above)
Comment (ckoopmann) [reply manifest.json:43]: This manifest is purely for ledger live purposes, I don't think it is related to the one in `public` (although I don't know what the lattter one is used fore)

Comment (ckoopmann) [reply manifest.json:43]: I moved it to `ledger_live/manifest.json` to make this distinction clearer.

File: package.json (hunk 2 above)
Comment (ckoopmann) [reply package.json:40]: Yes this replaces the `react-scripts` / create-react-app with `react-app-rewired` for the whole project.

We should maybe do some extra testing that none of the existing features break after this update.

Comment (ckoopmann) [reply package.json:40]: But yes I agree that this is only a temporary fix until we switch to next.js

File: src/utils/wagmi.ts (hunk 3 above)
Comment (ckoopmann) [reply src/utils/wagmi.ts:56]: Done

File: vercel.json (hunk 4 above)
Comment (ckoopmann) [reply vercel.json:8]: Good point. I will add the production eth-dapp browser address here additionally. (Keeping the old one for testing purposes)
